pyeudiw/tests/federation/test_static_trust_chain_validator.py

```diff
@@ -281,3 +286,97 @@
 def test_is_valid():
     assert StaticTrustChainValidator(
         trust_chain, [ta_jwk.serialize()]).is_valid
+
+
+invalid_intermediate = copy.deepcopy(intermediate_es)
+invalid_leaf_jwk = copy.deepcopy(leaf_jwk.serialize())
+invalid_leaf_jwk["kid"] = str(uuid.uuid4())
+
+invalid_intermediate["jwks"]['keys']  = [invalid_leaf_jwk]
+
+intermediate_signer = JWS(invalid_intermediate, alg="RS256",
+                          typ="application/entity-statement+jwt")
+invalid_intermediate_es_signed = intermediate_signer.sign_compact([intermediate_jwk])
+
+invalid_trust_chain = [
+    leaf_ec_signed,
+    invalid_intermediate_es_signed,
+    ta_es_signed
+]
+
+def test_is_valid_equals_false():
+    assert StaticTrustChainValidator(
+        invalid_trust_chain, [ta_jwk.serialize()]).is_valid == False
+    
+from unittest.mock import Mock
+    
+def test_retrieve_ec():
+    import pyeudiw.federation.trust_chain_validator as tcv_test    
```

Comment:
please move all the imports within the function in the top of the file

the imports must not be within the functions

Reply:
Resolved in [fix: moved imports](https://github.com/italia/eudi-wallet-it-python/pull/77/commits/293a7f29a4ed235c19103f09cc5e5d8e8e5d221f)